lib/index.js

```diff
@@ -260,12 +260,11 @@ export async function exec (argv2) { // eslint-disable-line complexity
   // targets
 
   const sTargets = argv.t || argv.target || argv.targets || '';
-  if (typeof sTargets !== 'string') {
-    throw wasReported(`Something is wrong near ${JSON.stringify(sTargets)}`);
-  }
 
   let targets = parseTargets(
-    sTargets.split(',').filter((t) => t)
+    Array.isArray(sTargets)
+      ? sTargets
+      : sTargets.split(',').filter((t) => t)
```

Comment:
do you want to trim any whitespace?

Reply:
Maybe not my place to say it. But have you considered working with [command-line-args](https://github.com/75lb/command-line-args). It can handle such things as multiple values for commands arguments and it has a brother project [command-line-usage](https://github.com/75lb/command-line-usage) that can generate the help documentation. It was a real time saver on my last project can't recommend it enough.